system_strings/sysmes_text.en

```diff
@@ -635,13 +635,13 @@ Akiha-sama Returns
 Greet her cheerfully.
 Greet her quietly.
 I had better start apologizing.
-A foolish brother ought to die.
-Insincere Suspicious Policemen
+Foolish brothers ought to die.
+Sleazy Policemen
 I'll ask what he wants.
 I'll ignore them.
 Slipping back into a routine.
 Do my utmost to intercept her.
-Do my utmost to focus on lesson.
+Try to focus on the lesson.
```

Comment:
I'd keep the pattern here, the choices in game do IIRC

Reply:
the problem was "focus on lesson". No space to add "the" so we had to reword the whole line